File: Project Oppenheimer (depriciated)/Project Oppenheimer/Card.cpp
#include "Card.h"

Card::Card(int i, int o, int a)
{
	id = i;
	opsValue = o;
	affiliation = a;
}

int Card::get_id()
{
	return this->id;
}


int Card::get_value()
{
	return this->opsValue;
}


int Card::get_affiliation()
{
	return this->affiliation;
}

File: Project Oppenheimer/Project Oppenheimer/Properties/Card.cs
namespace Project_Oppenheimer.Properties
{
    public struct Card
    {
        public string name;
        public int id;
        public int opsValue;
        public int affiliation;
        public bool scoring;
        public bool oneTimeEvent;

        public Card(string _name, int _id, int ops, int _affiliation, bool _scoring, bool OTE)
        {
            name = _name;
            id = _id;
            opsValue = ops;
            affiliation = _affiliation;
            scoring = _scoring;
            oneTimeEvent = OTE;
        }
    }
}
